core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/While.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2017 Crown Copyright
```

Comment:
Could you update your template for the copyright header so that the correct date is included?